modules/ldsg.js

```diff
@@ -0,0 +1,136 @@
+    const online = ldsg.members.cache.filter((member) => member?.presence && member.presence.status != "offline");
+    const response = `📈 **Members:**\n${ldsg.memberCount} Members\n${online.size} Online`;
+    await interaction.reply({ content: response });
+  } catch (error) { u.errorHandler(error, interaction); }
+}
+
+const sendOptions = [
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("suggestionPA").setEmoji("🗣️").setLabel("Public Affairs").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("suggestionLog").setEmoji("🚚").setLabel("Logistics").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("suggestionOps").setEmoji("🗓️").setLabel("Operations").setStyle(ButtonStyle.Success),
+    new u.Button().setCustomId("suggestionMgmt").setEmoji("⚙️").setLabel("Management").setStyle(ButtonStyle.Primary),
+    new u.Button().setCustomId("suggestionIcarus").setEmoji("🤖").setLabel("Icarus").setStyle(ButtonStyle.Primary),
+  ]),
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("suggestionIgnore").setEmoji("⚠️").setLabel("Ignore").setStyle(ButtonStyle.Danger),
+  ])
+];
+
+const replyOption = [
+  u.MessageActionRow().setComponents([
+    new u.Button().setCustomId("suggestionReply").setEmoji("🗨️").setLabel("Reply to user").setStyle(ButtonStyle.Primary)
+  ])
+];
+
+/** @param {Discord.ChatInputCommandInteraction} int */
+async function slashLdsgSuggest(int) {
+  const suggestion = int.options.getString("suggestion", true);
+  await int.deferReply({ ephemeral: true });
+  const embed = u.embed({ author: int.user })
+    .addFields({ name: "Suggestion:", value: suggestion })
+    .setFooter({ text: `User ID: ${int.user.id}` });
+  await int.client.getTextChannel(u.sf.channels.suggestionBox)?.send({ embeds: [embed], components: sendOptions });
+  int.editReply("Sent!");
+  return int.user.send({ content: "Sent:", embeds: [embed] });
+}
+
+/** @param {Discord.ButtonInteraction<"cached">} int */
+async function processCardAction(int) {
+  try {
+    const suggestion = int.message;
+    const embed = u.embed(suggestion.embeds[0]);
+    const modal = new u.Modal().addComponents(
+      u.ModalActionRow().addComponents([
+        new u.TextInput()
+          .setCustomId("text")
+          .setLabel("Reply")
+          .setStyle(Discord.TextInputStyle.Short)
+          .setRequired(true)
+          .setPlaceholder("Your reply to the user")
+      ])
+    ).setCustomId("reply").setTitle("Suggestion Reply");
+    let submitted;
+    let em;
+    let reply;
+    let member;
+    const channel = {
+      suggestionPA: u.sf.channels.publicaffairsForum,
+      suggestionLog: u.sf.channels.logisticsForum,
+      suggestionOps: u.sf.channels.operationsForum,
+      suggestionMgmt: u.sf.channels.managementForum,
+      suggestionIcarus: u.sf.channels.bottestingForum
+    }[int.customId];
+    if (channel) {
+      embed.addFields({ name: `Sent to <#${channel}>`, value: `by ${int.user}` });
```

Comment:
a cleaner solution might be to have the name be "Status" and the value be what you have as the current name and value